app/controllers/api/v1/statuses_controller.rb

```diff
@@ -48,11 +48,18 @@ def create
       scheduled_at: status_params[:scheduled_at],
       application: doorkeeper_token.application,
       poll: status_params[:poll],
+      safeguard_mentions: status_params[:safeguard_mentions],
       idempotency: request.headers['Idempotency-Key'],
       with_rate_limit: true
     )
 
     render json: @status, serializer: @status.is_a?(ScheduledStatus) ? REST::ScheduledStatusSerializer : REST::StatusSerializer
+  rescue PostStatusService::UnexpectedMentionsError => e
+    extra_accounts = ActiveModel::Serializer::CollectionSerializer.new(
+      e.extra_accounts,
+      serializer: REST::AccountSerializer
+    )
+    render json: { error: e.message, extra_accounts: extra_accounts }, status: 422
```

Comment:
`unexpected_accounts` might be better, though we have no convention for something like this which makes me uncomfortable.